trunk/SUAVE/Methods/Center_of_Gravity/compute_aircraft_center_of_gravity.py

```diff
@@ -0,0 +1,182 @@
+## @ingroup Methods-Center_of_Gravity
```

Comment:
Wait wait... did this script come back from hell? I thought I killed it!!! HOW IS IT BACK????!!!!??

Reply:
It's been replaced by vehicle.center_of_gravity()

Reply:
Sorry. this is an old code that I updated without realizing it. I did try your method and the CG is pushed way forward since the moment of individual components is 0 ( origin of passengers, avionics, etc is 0). In this script, I don't account for that and get a reasonable CG. Thoughts?